Detections/AzureDevOps/AzDOAdminGroupAdditions.yaml

```diff
@@ -0,0 +1,60 @@
+id: 89e6adbd-612c-4fbe-bc3d-32f81baf3b6c
+name: Azure DevOps Administrator Group Monitoring
+description: |
+  'This detection monitors for additions to project or project collection administration groups in an Azure DevOps Organization.'
+severity: High
+queryFrequency: 2h
+queryPeriod: 4h
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+  - Persistence
+  - Escalation
```

Comment:
Please use the MITRE ATT&CK full names with no spaces.  Here Escalation would be PrivilegeEscalation and we also require Techniques in Detections - https://attack.mitre.org/techniques/enterprise/

Format as follows -

relevantTechniques:
  - T1078
  - T1110

Reply:
fixed in next commit